src/globals.h

```diff
@@ -101,6 +101,44 @@ extern int vb6Round(double x);
  */
 extern double vb6Round(double x, int decimals);
 
+
+// utilities for stringindex_t
+extern std::vector<std::string> g_LevelString;
+extern size_t g_numWorldString;
+
+inline std::string GetS(stringindex_t index)
+{
+    if(index == STRINGINDEX_NONE)
+    {
+        return "";
```

Comment:
```cpp
return std::string();
```

Reply:
Are you saying that this is the proper style to construct a new empty string? (I don't think we'd be allowed to return a reference to a stack-constructed string, whether it's constructed `std::string` or by casting `""`.)

Reply:
Various static analyzers blame this case as it may. 

I meant, const reference, it bumps the life of temporary object, so:

```cpp
void call1(const std::string &one) {}
void call2(std::string &one) {}

call1(std::string()); // will work fine
call2(std::string()); // will fail at compilation
```

Reply:
Interesting, okay, then I will make this change and make it a const ref. Thanks for the tip!

Reply:
Wait, this isn't the case I'm dealing with here. You can't return a reference to a stack-allocated std::string() even if the reference is const, because the reference will be deallocated as soon as the function returns. I'll make a global const empty std::string instead.

Reply:
I see. Should be fine then.